worksheet/index.qmd

```diff
@@ -0,0 +1,152 @@
+
+Once you've picked a Post-it, open the issue and read the details in full. At this point, you might discover the issue is outside your wheelhouse; if so, no problem, just return the Post-it to the wall and try again.
+
+```{=typst}
+#colbreak()
+```
+
+```{=typst}
+#align(center)[
+  #box(
+    height: 3in,
+    width: 3in,
+    stroke: black,
+    [#align(center + horizon)[Place Post-it here]]
+  )
+]
+```
+
+```{=typst}
+#v(1in)
+```
+
+### 2. Get the source locally
+
+* **Fork** and clone the repository using this syntax: 
+    - `usethis::create_from_github("______________/_______________")` 
+    - (e.g `usethis::create_from_github("tidyverse/ggplot2")`)
+
+* `pak::local_install_dev_deps()` to make sure you've got the necessary packages.
+
+* Run `devtools::document()` to make sure you have everything necessary to update the docs before you make any changes.
+
+* If running `devtools::document()` fails or produces changes, something is probably off with your setup and you should seek help.
+
+```{=typst}
+#pagebreak()
+```
+
+### 3. Make the change
+
+* Create a new branch for your changes to live with
+  * `usethis::pr_init("______________________")`
+
+* Make your change. 
+  
+* use `devtools::document()` to update the docs.
+
+```{=typst}
+#set list(marker: [#scale(x: 200%, y: 200%, origin: bottom)[#sym.square.stroked]])
+```
+
+* Commit the change.
+  * Check changed files in the git pane.
+  * Press `Commit` button.
+  * Write a short message (<80 characters) detailing the change. use the [magic fixes keyword](https://help.github.com/en/articles/closing-issues-using-keywords), e.g. "Fixes #123", to link & close your issue.
+  * Press `Commit` button.
+
+```{=typst}
+#set list(marker: ([#scale(x: 200%, y: 200%, origin: bottom)[#sym.square.stroked]], [-]))
+```
+
+```{=typst}
+#v(1in)
+```
+
+### 4. Submit your PR
+
+Push and make a PR with `usethis::pr_push()`.
+  
+- Make the title descriptive:
+  - Not good: "Working on issue 45"
+  - Good: "Use stricter regexes for package and file name checks"
+  - Your title: `"_______________"`
+
+- Example with descriptive title and description containing magic keywords: https://github.com/r-lib/usethis/pull/742 
+
+- Write your PR number on the Post-it and move it to the review wall. 
+
+- Ring the gong to celebrate your successful submission!
+
+```{=typst}
+#v(0.5in)
+```
+
+### 5. Wait for review
+
+A person from the tidyverse team will try and review your issue as quickly as possible, reading through your PR and offering suggestions for how to improve it. If you want to work on something else while you wait, use `pr_pause()` to pause your pull request while you work on other stuff. It's possible the reviewer might make changes directly to your review - if that happens you can use `pr_pull()` to get their code back onto your computer.
```

Comment:
I think you need to mention `pr_resume()` after `pr_pause()`, but before `pr_pull()`. It's possible this is more readable as a mini bullet list?

* `pr_pause()` blah blah
* `pr_resume()` blah blah
* `pr_pull()` blah blah
* `pr_finish()` blah blah

They'd only need `pr_pause()` / `pr_resume()` if they plan to work on a different project in the same repo. But I know space is tight here.